net/openthread-br/Makefile

```diff
@@ -0,0 +1,95 @@
+# SPDX-FileCopyrightText: 2022 Stijn Tintel <stijn@linux-ipv6.be>
+# SPDX-License-Identifier: GPL-2.0-only
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openthread-br
+PKG_SOURCE_DATE:=2023-03-29
+PKG_SOURCE_VERSION:=cbeaf817c54ad06d0238d0fa2ec2a5b4b32ba805
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/openthread/ot-br-posix.git
+PKG_MIRROR_HASH:=b0968851538783dd6580be85e11257ceff5fe5ce8e5e60576cc7a05cde770524
+
+#PKG_BUILD_DIR:=$(BUILD_DIR)/ot-br-posix-thread-br-certified-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Stijn Tintel <stijn@linux-ipv6.be>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/openthread-br
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=OpenThread Border Router
+  DEPENDS:= \
+	+libblobmsg-json \
+	+libjson-c \
+	+libncurses \
+	+libreadline \
+	+libstdcpp \
+	+libubox \
+	+libubus \
+	+mdnsd \
+	+mdnsresponder
+endef
+
+define Package/openthread-br/description
+  A Thread border router for POSIX-based platforms.
+endef
+
+define Package/openthread-br/conffiles
+/etc/config/openthread
+/var/lib/thread
+endef
+
+CMAKE_OPTIONS += \
+	-DOT_BORDER_ROUTER:BOOL=ON \
+	-DOT_BORDER_ROUTING_NAT64:BOOL=ON \
+	-DOT_CHANNEL_MANAGER:BOOL=ON \
+	-DOT_CHANNEL_MONITOR:BOOL=ON \
+	-DOT_COMMISSIONER:BOOL=ON \
+	-DOT_ECDSA:BOOL=ON \
+	-DOT_FIREWALL:BOOL=OFF \
+	-DOT_SERVICE:BOOL=ON \
+	-DOT_SRP_CLIENT:BOOL=ON \
+	-DOT_SRP_SERVER:BOOL=ON \
+	-DOTBR_BACKBONE_ROUTER:BOOL=ON \
+	-DOTBR_BORDER_ROUTING:BOOL=ON \
+	-DOTBR_DNSSD_DISCOVERY_PROXY:BOOL=ON \
+	-DOTBR_DUA_ROUTING:BOOL=OFF \
+	-DOTBR_MDNS=mDNSResponder \
+	-DOTBR_OPENWRT:BOOL=ON \
+	-DOTBR_REST:BOOL=ON \
+	-DOTBR_SRP_ADVERTISING_PROXY:BOOL=ON \
+	-DOTBR_SRP_SERVER_AUTO_ENABLE:BOOL=ON \
+	-DOTBR_TREL:BOOL=ON
+
+TARGET_CFLAGS += -DOPENTHREAD_POSIX_CONFIG_DAEMON_SOCKET_BASENAME=\\\"/var/run/openthread-%s\\\"
+
+define Package/openthread-br/install
+	$(INSTALL_DIR) \
+		$(1)/etc/config $(1)/etc/init.d \
+		$(1)/usr/lib/lua/luci/controller/admin \
+		$(1)/usr/lib/lua/luci/view/admin_thread $(1)/usr/sbin \
+		$(1)/var/lib/thread \
```

Comment:
`$(1)/var/lib/thread \`
I had to remove this to get it to compile on 21.02.7

Reply:
This is targeted at master. If you want to use it on anything else, you're on your own.